### ISSUE TITLE
Some tweaks

### ISSUE THREAD

**Issue by Riajyuu** [fregante/github-clean-feed#1]
- [ ] restore all favicons
- [ ] groups also releases, tags, commits, etc. aka everthing
- groups by authors

**Comment by fregante** [comment]
1. Are you talking about these? <img width="170" alt="favicons" src="https://cloud.githubusercontent.com/assets/1402241/26663847/25900cf8-46c0-11e7-8849-f9d6f99f9bab.png">
2. It makes sense 👍 
3. Too complex

**Comment by Riajyuu** [comment]
>Are you talking about these?

Exactly.

As far as authors, maybe I should correct a little bit: from the same repos

**Comment by fregante** [comment]
> As far as authors, maybe I should correct a little bit: from the same repos

I'm not sure I understand, can you make an example?

**Comment by Riajyuu** [comment]
![image](https://cloud.githubusercontent.com/assets/16912240/26664695/193196ce-46c6-11e7-9450-8506be3026b6.png)


See this case:

Group news from ScreenToGif together
Another group are news from you-get

**Comment by fregante** [comment]
They are grouped like this now right?

- soimort/you-get
    - \#2054 rosynirvana commented
    - \#2047 rosynirvana commented
- NickeManarin/ScreenToGif
    - \#100 closed issue
    - \#100 commented

How should they be grouped?

**Comment by Riajyuu** [comment]
Yes, exactly. Another plausible situation:
* soimort/you-get
    * #2054 rosynirvana commented
    * #2047 rosynirvana commented
    * [owner] created tags ?????
* NickeManarin/ScreenToGif
    * #100 closed issue
    * #100 commented
    * [owner] pushed master ????

**Comment by fregante** [comment]
That's suggestion 2, but what is suggestion 3? ("groups by authors")

**Comment by Riajyuu** [comment]
They are a little bit different: Authors simple means the same User.
Aka, list activities from the same user at different places together.
e.g. : 
rosynirvana : 
* commented at you-get issue ???
* open pull request at ScreenToGif ???
* create repo ????

**Comment by fregante** [comment]
Yeah that's too complex because things are already grouped by repo, they can't be grouped by both things.

**Comment by Riajyuu** [comment]
> they can't be grouped by both things.

What about this: when this feature is enabled, **only** group things as users.

The benefit is that you can track users you followed in a more staightforward way.

**Comment by fregante** [comment]
The issue is that to do it I'd have to almost rewrite everything. You can group them by user with a simple change, but then each event is missing the repo name:

```diff
diff --git a/src/content.js b/src/content.js
index 6cac979..8f14d64 100644
--- a/src/content.js
+++ b/src/content.js
@@ -43,10 +43,10 @@ function groupByRepo(events) {
 			relatedEl,
 			timeEl: originalEvent.querySelector('.time'),
 		};
-		if (!repos.has(repo)) {
-			repos.set(repo, new Set());
+		if (!repos.has(actorEl.textContent)) {
+			repos.set(actorEl.textContent, new Set());
 		}
-		repos.get(repo).add(event);
+		repos.get(actorEl.textContent).add(event);
 		originalEvent.remove();
 		return repos;
 	}, new Map());
```

<img width="386" alt="grouped by actor" src="https://cloud.githubusercontent.com/assets/1402241/26665245/d8afeb00-46ca-11e7-8912-e9727410ae4a.png">

This is too much work and it's not useful to me.

**Comment by Riajyuu** [comment]
Got it. Thanks for patience.
